src/app/login/login.component.scss

```diff
@@ -3,3 +3,21 @@
   margin-bottom: 50px;
   float: left;
 }
+
+.login-card {
+  max-width: 200px;
+  margin-bottom: 100px;
+  margin-right: 200px;
+  margin-top: 50px;
+  float: right;
+}
```

Comment:
Since you said this is temporary, can you add a TODO or a FIXME so that we don't miss it? I don't recommend using `float` anywhere though.

Reply:
TODO added.